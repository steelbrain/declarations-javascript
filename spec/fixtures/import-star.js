import * as Helpers from 'some-helpers'

Helpers.test()
