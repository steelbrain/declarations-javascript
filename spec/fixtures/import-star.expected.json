[
  {
    "name": null,
    "position": {
      "start": {
        "line": 1,
        "column": 25
      },
      "end": {
        "line": 1,
        "column": 39
      }
    },
    "source": {
      "name": null,
      "filePath": "some-helpers",
      "position": {
        "start": {
          "line": 1,
          "column": 25
        },
        "end": {
          "line": 1,
          "column": 39
        }
      }
    }
  },
  {
    "name": "Helpers",
    "position": {
      "start": {
        "line": 3,
        "column": 0
      },
      "end": {
        "line": 3,
        "column": 7
      }
    },
    "source": {
      "name": null,
      "filePath": null,
      "position": {
        "start": {
          "line": 1,
          "column": 12
        },
        "end": {
          "line": 1,
          "column": 19
        }
      }
    }
  }
]